Documentation/Books/Manual/Administration/Configuration/README.md

```diff
@@ -0,0 +1,280 @@
+# Configuration
+
+The [programs and tools](../../Programs/README.md) shipped in an
+ArangoDB package can be configured with various _startup options_.
+
+- Startup options you specify on a command line are referred to as
+  [command line options](#command-line-options):
+
+  `arangosh --server.database myDB`
+
+- The same options can also be set via
+  [configuration files](#configuration-file-format),
+  using a slightly different syntax:
+
+  `server.database = myDB`
+
+- There are also _flags_ which are for command line usage only,
+  such as `‑‑help` and `‑‑version`. They don't take any value
+  in contrast to options.
+
+Find the available options and flags in the _Options_ sub-chapters of the
+respective [Programs & Tools](../../Programs/README.md) sub-chapter, like the
+[ArangoDB Server Options](../../Programs/Arangod/Options.md).
+
+The [ArangoDB Starter](../../Programs/Starter/README.md) works differently
+to the other programs and tools. It uses `setup.json` files for its own
+[configuration](../../Programs/Starter/Architecture.md#starter-data-directory)
+and has a fluent command line interface to execute certain actions.
```

Comment:
Need to explain that the starter generates .conf files so the shipped ones won't be used if you use starter.